scripts/vagrant-provision-debian.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+apt-get update -y
+
+#install build utils
+apt-get install -y build-essential
+
+#install basic utils
+apt-get install -y nano curl git unzip mongodb-server
+
+#install node
+curl -sL https://deb.nodesource.com/setup_7.x | bash -
+apt-get install -y nodejs
+
+#install yarn
+curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
+echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
+apt-get update -y && apt-get install -y yarn
+
+#sox install for sonus speech recognition
+apt-get install -y sox libsox-fmt-all alsa-utils
+
+apt-get install -y libatlas-base-dev libatlas3gf-base
+apt-get install -y usbutils
+apt-get install -y lirc
+apt-get install -y libzmq-dev
+
```

Comment:
As it's for development purpose we can add here all plugins of the lisa organisation and do a yarn install on them what do you think ? It will speed up full setup and allow others to add easily there own plugins

Reply:
I agree, might we consider using git submodules for the plugins?  It's a little used feature and some people don't like it though.

Reply:
I don't like it to be honest lol :D even more here because I don't want plugin to be "link" to the main app. I prefer having an array of plugins and iterate on it to clone repos. What do you think?